src/components/BlockNativeOnboard/index.tsx

```diff
@@ -38,10 +39,17 @@ const OnboardButton = styled.button`
 export function OnboardConnectButton(): JSX.Element {
     //eslint-disable-next-line @typescript-eslint/no-unused-vars
     const [{ wallet, connecting }, connect, disconnect] = useConnectWallet()
+    const sendData = useSendAnalyticsData()
+
+    const onWalletConnect = async () => {
+        await connect()
```

Comment:
wallet_connect_start means we started connecting. in such case  I don't think we have to await

Reply:
done